### PR TITLE
273 Add grouped Runs navigation

### PR DESCRIPTION
**Pull request by GeorgiChochov**

### CODE REVIEW

File: menas/src/main/scala/za/co/absa/enceladus/menas/repositories/RunMongoRepository.scala [hunk 1]
```diff
@@ -193,6 +193,94 @@ class RunMongoRepository @Autowired()(mongoDb: MongoDatabase)
...
+        Document("""{latestRunStartDateTime: {
+                   |  $dateToString: {
+                   |    date: "$latestStart",
+                   |    format: "%d-%m-%Y %H:%M:%S %z",
+                   |    timezone: "$timezone"
+                   |  }
+                   |}},""".stripMargin),
+        excludeId()
+      )),
+      sort(ascending("datasetVersion"))
```
Comment (yruslan) [inline menas/src/main/scala/za/co/absa/enceladus/menas/repositories/RunMongoRepository.scala]: I'm wondering if descending order here is more convenient?

Comment (GeorgiChochov) [reply menas/src/main/scala/za/co/absa/enceladus/menas/repositories/RunMongoRepository.scala]: Makes sense

File: menas/ui/components/run/runDatasetNameMaster.view.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<core:View id="runMasterView" xmlns:core="sap.ui.core" xmlns="sap.m"
+           controllerName="components.run.runDatasetNameMaster"
+           xmlns:cust="http://schemas.sap.com/sapui5/extension/sap.ui.core.CustomData/1">
+    <Page id="masterPage" title="Dataset Runs" showNavButton="true" navButtonPress="onPressMasterBack">
```
Comment (HuvarVer) [inline menas/ui/components/run/runDatasetNameMaster.view.xml]: I think there should be the same arrows ( `>` ) as are at the Navigation menu. For better understanding that it rolled out after pressing the button.

Comment (HuvarVer) [reply menas/ui/components/run/runDatasetNameMaster.view.xml]: image
image

Comment (GeorgiChochov) [reply menas/ui/components/run/runDatasetNameMaster.view.xml]: That's an excellent point! Thanks

File: menas/ui/components/app.controller.js [hunk 3]
```diff
@@ -92,7 +94,19 @@ sap.ui.define([
...
+      this._app.toMaster(this.createId("runsDatasetNamePage"));
+    },
+
+    onRunsDatasetNamePress: function (channel, event, data) {
+      this._eventBus.publish("runs", "list-grouped-version", data);
+      this._app.toMaster(this.createId("runsDatasetVersionPage"));
+    },
+
+    onRunsDatasetVersionPress: function (channel, event, dataset) {
+      console.log(dataset)
```
Comment (Zejnilovic) [inline menas/ui/components/app.controller.js]: Is this `console.log` intentional? If so, it is missing a `;`

Comment (GeorgiChochov) [reply menas/ui/components/app.controller.js]: Hah, nope, missed it

**Pull request merged**